packages/patternfly-4/react-core/src/components/Nav/Nav.docs.js

```diff
@@ -0,0 +1,35 @@
+import { Nav, NavGroup, NavList, NavItem, NavItemSeparator, NavExpandable, NavVariants } from '@patternfly/react-core';
```

Comment:
This file should not exist. No more *.docs.js files. Examples are in markdown file